libs/sdk-ui-dashboard/styles/scss/filterBar.scss

```diff
@@ -128,6 +128,10 @@ $attribute-filter-drag-handle-left: 10px;
     .dash-filters-date {
         position: relative;
 
+        .gd-attribute-filter-dropdown-button__next {
+            display: flex;
```

Comment:
I'm wondering what effect this would have, it would affect all attribute filters, right? Does it have some relation to any flex- features in the error states you're adding?

Reply:
Thanks, hopefully there will be no hidden side-effects.